### PR TITLE
Totp reset instructions

### PR DESCRIPTION
**Pull request by wjwoodson**
Add instructions for cloud.gov operators to reset TOTP tokens/authenticators for users authenticating via cloud.gov.

### CODE REVIEW

**Review by jmcarp — DISMISSED** [review]

File: content/docs/ops/managing-users.md [hunk 1]
```diff
@@ -27,6 +27,28 @@ If they log in with a cloud.gov account that has its own password (including `OR
 uaac target uaa.cloud.gov
 ```
 
+## Resetting TOTP tokens
+
+Follow the process below for users logging in with a cloud.gov account. It is our responsibility to verify the reset request is legitimate.
+
```
Comment (pburkholder) [inline content/docs/ops/managing-users.md]: @brittag mentioned:
> You just have to check whether the request makes sense on a basic level; for example, if we know from a customer team directly that we should be expecting a crop of legitimate requests.

Comment (wjwoodson) [reply content/docs/ops/managing-users.md]: 👍

File: content/docs/ops/managing-users.md [hunk 2]
```diff
@@ -27,6 +27,28 @@ If they log in with a cloud.gov account that has its own password (including `OR
...
+## Resetting TOTP tokens
+
+Follow the process below for users logging in with a cloud.gov account.
+
+If the user requesting a reset has any apps, routes, or services in their sandbox or access to any other spaces or orgs make sure they **[are informed]({{< relref "docs/getting-started/accounts.md#if-you-can-t-access-your-token-codes" >}})** these will be removed.
+
+1. Remove all apps, routes, and services from the user's sandbox.
+2. Remove user's permissions to all [spaces and orgs](https://docs.cloudfoundry.org/adminguide/cli-user-management.html#orgs-spaces) other than their sandbox.
+
+    For those spaces and orgs, notify the Space Managers and Org Managers that we've removed the user's access because of their request to reset their account's authentication application.
```
Comment (brittag) [inline content/docs/ops/managing-users.md:39]: Enhancement for the future: we should have little template email snippets here + guidance on how to contact them. Probably just copying them into the support thread via Zendesk makes sense.

**Review by brittag — DISMISSED** [review]
Looks good to me from a policy/procedure perspective - thanks for documenting this! I'll leave this un-merged since I'm not sure whether it needs additional technical review.

**Comment by wjwoodson** [comment]
@pburkholder after pairing on these this morning do you think we're good to merge?

File: content/docs/ops/managing-users.md (hunk 2 above)
Comment (mogul) [reply content/docs/ops/managing-users.md:39]: We could also have a script that automates all of this, but I'm hopeful login.gov will go in the lava soon so we won't be dealing with token resets long enough for that to be useful...!

**Review by mogul — APPROVED** [review]

**Pull request merged**